docs/source/ui/pages/table.md

```diff
@@ -0,0 +1,42 @@
+## Table
+
+### Overview
+
+The table helps to explore and compare training runs results, logged hyper-parameters, system info, etc. As well as to apply quick actions (hide, delete, archive, etc.) over a batch of selected items within a few clicks.
+
+### Delete a batch of items
+
+To delete a batch of runs select rows and click on the `Delete` button, which appears at the top of the table.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC; height: 200px" alt="Delete a batch of items" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/delete_a_batch_of_items.png">
+
+.. note::
+In-progress runs can't be deleted. Wait until the process is finished or terminate the process to be able to delete them.
+
+After clicking on the `Delete` button a popup appears where `finished` and `in-progress` runs are divided into separate tables.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC;" alt="Delete a batch of items popup" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/delete_a_batch_of_items_popup.png">
+
+To undo a selection, click on the X icon which appears when a row is hovered over.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC;" alt="Delete a batch of items popup remove run" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/delete_a_batch_of_items_popup_remove_run.png">
+
+### Archive/Unarchive a batch of items
+
+To archive/unarchive a batch of runs select rows and click on the `Archive/Unarchive` button, which appears at the top of the table.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC; height: 200px" alt="Archive a batch of items" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/archive_a_batch_of_items.png">
+
+After clicking on the `Archive/Unarchive` button a popup appears, where `archived` and `unarchived` runs are divided into separate tables.
+
+<img style="border-radius: 8px; border: 1px solid #E8F1FC" alt="Archive a batch of items popup" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/table/archive_a_batch_of_items_popup.png">
+
+To undo a selection, click on the X icon which appears when a row is hovered over.
```

Comment:
To undo the selection, hover over the selected row and click on the X button (appears on hover).